test/test_clicommand.py

```diff
@@ -58,7 +71,7 @@ def test_scan_facts_default(self):
                                            "--reset", "--reportfile",
                                            "data/test_report.csv", "--facts",
                                            "default", "ansible_forks",
-                                           "100"])
+                                           "100", "--vault", "./vault_pass"])
```

Comment:
I've moved all the files for testing to `/tmp`. Hopefully this is what you were going for, if not lets capture further updates here: https://github.com/quipucords/rho/issues/33